### PR TITLE
Monitor router status

### LINKED ISSUE #268: Monitor router status
Sometimes routers are turned off, plugged out or disconnect for some reason. It would be a useful feature to monitor their statuses through the site (we already have scripts on the server) and get notification when something goes wrong.

### PR DESCRIPTION
**Pull request by luksan47**
Fixes #268 
Internet admins will also be notified on the second failure via email.

### CODE REVIEW

File: app/Role.php [hunk 1]
```diff
@@ -65,6 +65,11 @@ public function object()
         return $this->possibleObjects()->where('id', $this->pivot->object_id)->first();
     }
 
+    public static function getUsers(string $roleName)
+    {
+        return Role::where('name', $roleName)->first()->users;
+    }
+
```
Comment (kdmnk) [inline app/Role.php:72]: What is this for?

Comment (kdmnk) [reply app/Role.php:72]: FYI: you can use firstWhere function

**Review by kdmnk — APPROVED** [review]
Nice. Will we have to add the ip addresses to the database manually?

**Comment by luksan47** [comment]
> Will we have to add the ip addresses to the database manually?
Yes, this will be part of the release. Opened #361 for this.

**Pull request merged**